doc/programming/parsing.rst

```diff
@@ -314,7 +314,7 @@ constraints on their values. ``EXPR`` must a be boolean expression
 that will be evaluated after the parsing for the field has finished,
 but before any hooks execute. If ``EXPR`` returns ``False``, the
 parsing process will abort with an error, just as if the field had
-been unparseable in the first place (incl. executing any :ref:`%error
+been untraceable in the first place (incl. executing any :ref:`%error
```

Comment:
Was this change intentional? I think I understood _unparseable, but _untraceable_ would appear here the only time in the user-searchable docs, and at least for me means something different.